cypress/integration/router_spec.js

```diff
@@ -0,0 +1,229 @@
+import { c, a } from '../support/utils.js';
+
+describe('Create event', () => {
+  beforeEach(() => {
+    cy.resetDb();
+    cy.cachedLogin();
+  });
+
+  // Open the hambuermenu and select by name, then assert by path
+  const openMenuAndSelect = (name, path) => {
+    cy.get(c('buttonGroup')).within(() => {
+      cy.get(c('searchIcon')).click();
+    });
+    cy.get(c('Search__navigationFlex')).within(() => {
+      cy.contains(name).click();
+    });
+    cy.url().should('contain', path);
+  };
+
+  it('should be able to navigate to events', () => {
+    cy.visit('/');
+    cy.get(c('navigation')).within(() => {
+      cy.contains('Arrangementer').click();
+    });
+    cy.url().should('contain', '/events');
+    cy.contains('Denne uken');
+    cy.contains('Liste');
+  });
+
+  it('should be able to navigate to joblistings', () => {
+    cy.visit('/');
+    cy.get(c('navigation')).within(() => {
+      cy.contains('Karriere').click();
+    });
+    cy.url().should('contain', '/joblistings');
+    cy.contains('Jobbannonser');
+    cy.contains('Søknadsfrist');
+  });
+
+  it('should be able to navigate to about-page', () => {
+    cy.visit('/');
+    cy.get(c('navigation')).within(() => {
+      cy.contains('Om Abakus').click();
+    });
+    cy.url().should('contain', '/pages/info-om-abakus');
+    cy.contains('Generelt');
+    cy.contains('Komiteer');
+  });
+
+  it('should be able to navigate to users profile', () => {
+    cy.visit('/');
+    cy.get(c('buttonGroup')).within(() => {
+      cy.get('img[alt="user"]').click();
+    });
+
+    // Go to profile
+    cy.get(c('Dropdown')).within(() => {
+      cy.get(a('/users/me')).click();
```

Comment:
I don't know how to switch out this with a contains, as the name is "webkom" or whatever, so I think it has to stay like this :/